packages/odyssey-storybook/src/components/odyssey-mui/Tabs/Tabs.stories.tsx

```diff
@@ -168,3 +184,52 @@ export const Icons: StoryObj<TabItemProps> = {
     selectTab({ canvasElement, step })("Tab Icon", "Xenomorphs");
   },
 };
+
+export const Controlled: StoryObj<TabItemProps> = {
+  render: function C() {
+    const [value, setValue] = useState("planets");
+
+    const onChange = (_e: unknown, value: string) => {
+      setValue(value);
+    };
```

Comment:
**NON-BLOCKING**

Can we also change this to be `onChange: TabsProps["onChange"]`? People often copy-paste out of stories, so I wanna make sure we set them up with the correct types; that's all.